<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralForm</class>
 <widget class="QWidget" name="GeneralForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="generalGroup">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="generalLayout">
          <item>
           <widget class="QLabel" name="transLabel">
            <property name="toolTip">
             <string>The translation may not load until qTox restarts.</string>
            </property>
            <property name="text">
             <string>Language:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="transComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>The translation may not load until qTox restarts.</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="generalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="startupLayout">
          <item>
           <widget class="QCheckBox" name="cbAutorun">
            <property name="toolTip">
             <string>Start qTox on operating system startup (current profile).</string>
            </property>
            <property name="text">
             <string>Autostart</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkUpdates">
            <property name="text">
             <string>Check for updates</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="cbSpellChecking">
            <property name="text">
             <string>Spell checking</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showSystemTray">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Show system tray icon</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <item>
             <widget class="QCheckBox" name="startInTray">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string comment="toolTip for Start in tray setting">qTox will start minimized
                                               in tray.
                                           </string>
              </property>
              <property name="text">
               <string>Start in tray</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="minimizeToTray">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string comment="toolTip for minimize to tray setting">After pressing
                                               minimize (_) qTox will minimize itself to tray,
instead of system taskbar.</string>
              </property>
              <property name="text">
               <string>Minimize to tray</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="closeToTray">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string comment="toolTip for close to tray setting">After pressing close (X)
                                               qTox will minimize to tray,
instead of closing itself.</string>
              </property>
              <property name="text">
               <string>Close to tray</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="themeGroupBox">
       <property name="title">
        <string>Theme</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item row="3" column="0">
         <widget class="QLabel" name="styleLabel">
          <property name="text">
           <string>Style:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="styleBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="themeColorLabel">
          <property name="text">
           <string>Theme color:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="themeColorCBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="timestamp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="timeExample">
            <property name="text">
             <string notr="true">Time Example</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="timestampLabel">
          <property name="text">
           <string>Timestamp format:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="dateFormats">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dateExample">
            <property name="text">
             <string notr="true">Date Example</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="dateformatLabel">
          <property name="text">
           <string>Date format:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>transComboBox</tabstop>
  <tabstop>cbAutorun</tabstop>
  <tabstop>checkUpdates</tabstop>
  <tabstop>showSystemTray</tabstop>
  <tabstop>startInTray</tabstop>
  <tabstop>minimizeToTray</tabstop>
  <tabstop>closeToTray</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>showSystemTray</sender>
   <signal>toggled(bool)</signal>
   <receiver>closeToTray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showSystemTray</sender>
   <signal>toggled(bool)</signal>
   <receiver>minimizeToTray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showSystemTray</sender>
   <signal>toggled(bool)</signal>
   <receiver>startInTray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
